### PR TITLE
[Fix] bug in check limits

### PR DESCRIPTION
**Pull request by jordiandreu**
It fails to check limits containing the value 0.

### CODE REVIEW

**Comment by jordiandreu** [comment]
@beteva I propose this change to protect the limits from values like None or NaN.

**Comment by beteva** [comment]
Well done, thanks

File: HardwareObjects/abstract/AbstractActuator.py [hunk 1]
```diff
@@ -142,6 +143,7 @@ def update_limits(self, limits=None):
         if limits is None:
             limits = self.get_limits()
 
-        if all(limits):
+        # All values are not None nor NaN
```
Comment (rhfogh) [inline HardwareObjects/abstract/AbstractActuator.py:146]: @beteva @jordiandreu 
Actually, what happens if one of the limits is zero? I thought someone mentioned that in the discussion

Comment (jordiandreu) [reply HardwareObjects/abstract/AbstractActuator.py:146]: If one of the limit values was 0, the nominal limits were not updated and no limitsChange signal were emitted.

**Pull request merged**